cves/CVE-2015-8218.yml

```diff
@@ -90,14 +97,12 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: d4a731b84a08f0f3839eaaaf82e97d8d9c67da46
-  note: ''
-- commit: ''
-  note: ''
+  note: 'Main fix commit'
+- commit: 'a7bbb7fb884a02a7b380ef7afa787fca756b9d82'
+  note: 'Cherry picked commit for version 2.8.2'
 vccs:
 - commit: b2e95e012c3c857eb1f8fddc7a41c0c4f0581f30
-  note: Identified by archeogit,
-- commit: ''
-  note: ''
+  note: Identified by archeogit
```

Comment:
```suggestion
```

Snipped, in case the previous comment above is merged. If not, then disregard this change.